docusaurus/docs/reactnative/common-content/core-components/channel/props/is_message_equal.mdx

```diff
@@ -0,0 +1,11 @@
+Custom function which accepts previous and next attachment and return boolean according to user needs.
+This is useful to implement custom logic for memoization. This is helpful when using custom properties in attachment object.
```

Comment:
```suggestion
return true if passing nextMessage to render would return the same result as passing prevMessage to render, otherwise return false.
```